design/one-pager-stack-ui-metadata.md

```diff
@@ -0,0 +1,155 @@
+# Stack Package UI configuration metadata
+
+* Owner: Steven Rathbauer (@rathpc), Marques Johansson (@displague)
+* Reviewers: Crossplane Maintainers
+* Status: Draft
+
+## Terms
+
+* **Stack**: A Stack is a Crossplane extension-manager managed package.  These packages may bundle one or many Crossplane applications or infrastructure provider controllers.
+* **Stacks**: Stacks could refer to more than one Crossplane extension-manager packages or the Extension-Manager system as a whole.
+
+See the [Stacks Design Doc](design-doc-stacks.md) for more details about Crossplane Stacks.
```

Comment:
this PR should probably also update the main design doc with a short description of this new UI focused addition to the stack package format

Reply:
Added in 13b4da9